manifests/c/ChemTableSoftware/AutorunOrganizer/5.19.0.0/ChemTableSoftware.AutorunOrganizer.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: ChemTableSoftware.AutorunOrganizer
+PackageVersion: 5.19.0.0

```

```suggestion
PackageVersion: 5.19
```
This must match Control Panel.